ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -571,6 +571,49 @@ def Conv(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Ngra
     return make_convolution_op(onnx_node, ng_inputs)
 
 
+def ConvTranspose(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
+    """Calculate convolution transpose."""
+    if len(ng_inputs) == 3:
+        data, weights, bias = ng_inputs
+    elif len(ng_inputs) == 2:
```

Comment:
What if number of inputs will be different? Consider adding `else` statement and give some warning/error.

Reply:
Usually we are not verify number of inputs and standard says there can be only 2 or 3 inputs.